addons/models/1000-Hadoop/patches/018-kafka_topic_add_rf_attribute.json

```diff
@@ -0,0 +1,23 @@
+{
+  "patches": [
+    {
+      "id": "TYPEDEF_PATCH_1000_018_001",
+      "description": "Add 'replicationFactor' attribute to kafka_topic",
+      "action": "ADD_ATTRIBUTE",
+      "typeName": "kafka_topic",
+      "applyToVersion": "1.7",
+      "updateToVersion": "1.8",
+      "params": null,
+      "attributeDefs": [
+        {
+          "name": "replicationFactor",
```

Comment:
can you also add this new attribute to the base 'kafka_topic' model - 1070-kafka_model.json?